glotaran/model/dataset_model.pyi

```diff
@@ -30,9 +34,9 @@ class DatasetModel:
     ) -> Generator[tuple[Parameter | int | None, Megacomplex | str], None, None]: ...
     def get_model_dimension(self) -> str: ...
     def finalize_data(self, dataset: xr.Dataset) -> None: ...
-    def overwrite_model_dimension(self, model_dimension: str) -> None: ...
+    def replace_model_dimension(self, model_dimension: str) -> None: ...
```

Comment:
`set_model_dimension` would fit better

Reply:
Fixed in [57c32a9](https://github.com/glotaran/pyglotaran/pull/983/commits/57c32a9b7c12ea41a18862f8c621615b3dcc0a80)